0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,82 +1,91 @@
+
+A single Interledger packet may traverse any number of connectors on its way to the destination. For efficiency reasons, each connector seeks to forward the packet on the shortest known path to the destination. See [IL-RFC 10](../0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md) for more information on routing in Interledger.
+
+### Senders
+
+A *sender* is a system which sends Interledger packets in order to:
 
-A *connector* is a system that facilitates *payments* across different ledgers. Connectors generate revenue from Interledger payments while accepting some risk.
+1. Transfer value from itself to a receiver or
+2. Exchange value from one type to another.
 
-In the Interledger model, connectors are described as separate logical systems even though the same entity may operate a ledger and a connector.
+When transferring value, the sender will first request payment details from the receiver, including the receiver's ILP address. Once it has those details, it will start sending Interledger packets using this ILP address as the destination. A large payment may be split into many smaller pieces and reassembled by the receiver. For more information on the overall payment process, see the section [Interledger Protocol Suite](#interledger-protocol-suite).
 
-A connector receives a local transfer on one ledger in exchange for making another local transfer on a different ledger. A single interledger payment may include multiple connectors and may traverse any number of ledgers.
+A sender may also exchange assets by acting as both sender and receiver. In this case, it would send one asset and receive another, effectively exchanging one for the other.
 
-If the ledgers represent different assets, the connectors set the exchange rate between the transfers. Connectors may generate revenue from the difference in value between incoming and outgoing transfers. Senders may request quotes from multiple connectors to determine the best price before sending a payment.
+Senders typically choose a connector and send all their traffic through this connector. If they are at any point dissatisfied with their connector, they may switch to a different one. Sophisticated senders may operate a connector of their own and maintain relationships with multiple connectors to minimize costs and maximize availability.
 
-Connectors *peer* with one another to exchange information used to determine the best route for a payment.
+### Receivers
+
+A *receiver* is a system which accepts incoming Interledger packets and provides a cryptographic receipt for each packet which is returned to the sender along the same path. For more information on the cryptographic exchange, see [Payment Flow](#payment-flow).
 
 **Interledger ensures that senders' funds are safe throughout an multi-hop payment and cannot be stolen by faulty or malicious connectors (see [Interledger Security](#interledger-security)).**
```

Comment:
This safety now comes from the small chunk size, right? Since transfers now go over accounts, not ledger, a faulty or malicious connector *can* steal a chunk, but only one, and when it does steal a chunk, it would immediately lose traffic to competitors.

Reply:
No, the small chunk size quantitatively reduces the risk, it does not change the nature of the risk.

Before: **Everyone has to trust their direct neighbors, Interledger ensures you do not have to trust any other intermediaries.** Theoretically, these were meant to always be ledgers you already have to trust. But, in practice, we started using trustlines which didn't really fit the ledger model at all.

Now: **Everyone has to trust their direct neighbors, Interledger ensures you do not have to trust any other intermediaries.** (The same.) The only difference is that an escrowing connector (formerly known as a ledger) is now only one of several possible constellations. At each hop (account), one side needs to take risk.

You might say: But before, I only had to trust the ledger, which could be a decentralized blockchain. Well, connectors could be decentralized blockchains, too. The reduction in risk is just likely not worth the increase in complexity/cost, so I don't think decentralized connectors will play a big role anytime soon.

Reply:
So then changing it to "cannot be stolen by faulty or malicious *intermediate* connectors" would make the statement true. Maybe we want to give a special name to the first connector, e.g. "entry node", to distinguish its special role in the trust model from that of intermediate nodes? cc @dappelt

Reply:
i'll propose to use the word 'intermediaries' in my continuation of this PR. we should probably revise the glossary and add a few clarifying notes there about what a ledger is, what a connector is, and what a 'node'/ILSP/neighbor is, once this is merged.